packages/core/src/views/account-center/Maximized.svelte

```diff
@@ -231,35 +225,38 @@
 
       <!-- actions -->
       <div class="actions flex flex-column items-start">
-        <!-- connect another wallet -->
-        <div
-          on:click={() => connect()}
-          class="action-container flex items-center pointer"
-        >
-          <div class="plus-icon flex items-center justify-center">
-            {@html plusCircleIcon}
-          </div>
-          <span class="action-text"
-            >{$_('accountCenter.connectAnotherWallet', {
-              default: en.accountCenter.connectAnotherWallet
-            })}</span
+        <!-- Hide for Mobile  -->
+        {#if innerWidth > 480}
```

Comment:
Would it be more robust to use our device check function with `bowser` to detect if on mobile?

Reply:
Replaced it with the bowser mobile detection.